src/libstd/sys/windows/stdio.rs

```diff
@@ -1,17 +1,16 @@
 #![unstable(issue = "0", feature = "windows_stdio")]
 
 use crate::char::decode_utf16;
-use crate::cmp;
 use crate::io;
-use crate::ptr;
-use crate::str;
+use crate::{ptr, cmp, str};
 use crate::sys::c;
 use crate::sys::cvt;
 use crate::sys::handle::Handle;
 
 // Don't cache handles but get them fresh for every read/write. This allows us to track changes to
 // the value over time (such as if a process calls `SetStdHandle` while it's running). See #40490.
 pub struct Stdin {
+    #[allow(dead_code)]
```

Comment:
I think this and the other changes in this file may not be necessary any more?

Reply:
That seems likely indeed, thanks for noticing!

Reply:
Fixed